docs/DeveloperGuide.md

```diff
@@ -581,6 +582,27 @@ testers are expected to do more *exploratory* testing.
 
 2. _{ more test cases …​ }_
 
+## **Prefix Summary**
+
+| Prefix    | Meaning | Constraints                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | 
+|-----------|---------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `n/`      | Name of contact | Can only contain alphanumeric characters and selected symbols like `/`, `-` and `'`.                                                                                                                                                                                                                                                                                                                                                                                                       |
+| `p/`      | Phone number of contact | Only numbers are allowed and must contain at least 3 digits. Only used in `addc` and `editc` commands.                                                                                                                                                                                                                                                                                                                                                                                     | 
```

Comment:
May be confusing that there are two separate entries for `p/`... maybe best to combine the two into a row and explain in the cells